project/AppDependencies.scala

```diff
@@ -2,13 +2,13 @@ import sbt._
 
 object AppDependencies {
 
-  val hmrcTestVersion = "3.3.0"
-  val scalaTestVersion = "3.0.5"
+  val hmrcTestVersion = "3.6.0-play-25"
+  val scalaTestVersion = "3.0.6"
   val scalatestplusVersion = "2.0.1"
-  val mockitoVersion = "2.23.4"
+  val mockitoVersion = "2.25.0"
   val pegdownVersion = "1.6.0"
```

Comment:
Just a question - can we remove pegdown?

Reply:
yes